src/main/java/edu/umich/tl/CourseDelete.java

```diff
@@ -169,69 +178,70 @@ private static void checkForApiType(Properties properties) {
 				} 
 			}
 		} catch (JsonParseException e) {
-			M_log.error("getPreviousTerms() has JsonParseException",e);
+			M_log.error("getTerms() has JsonParseException",e);
 		} catch (JsonMappingException e) {
-			M_log.error("getPreviousTerms() has JsonMappingException",e);
+			M_log.error("getTerms() has JsonMappingException",e);
 		} catch (IOException e) {
-			M_log.error("getPreviousTerms() has IOException",e);
+			M_log.error("getTerms() has IOException",e);
 		}catch (ParseException e) {
-			M_log.error("getPreviousTerms() has ParseException",e);
+			M_log.error("getTerms() has ParseException",e);
 		}
 		return enrollmentTerms;
 	}
 	/*
 	 * This method returns the List of terms for deletion of unused and unpublished courses. 
 	 */
-	private static ArrayList<String> getPreviousTerms(ArrayList<Term> terms) {
+	private static ArrayList<Term> getPreviousTerms(ArrayList<Term> terms) {
```

Comment:
I know, but I needed termName info to be displayed in logs when their are no un published courses. The list of Strings it was holding was canvasId and nobody knows  about them. I could made it a hashmap but i just felt just return List<terms> as it has more info to be used later for sending emailing and stuff